ykbuild/build.rs

```diff
@@ -136,8 +136,13 @@ fn main() {
         .current_dir(build_dir.as_os_str().to_str().unwrap());
 
     let mut build_cmd = Command::new("cmake");
+    let mut args = vec!["--build".into(), ".".into()];
+    if let Ok(jobs) = env::var("CARGO_JOBS") {
```

Comment:
I'd an idiot.

I saw that both `CARGO_BUILD_JOBS` and `NUM_JOBS` are supported and... merged the two in my head? I don't know which of the two to prefer. Any thoughts?

Reply:
Aha, setting `CARGO_BUILD_JOBS` causes cargo to set `NUM_JOBS` in the build script! Adjusting now.

Reply:
Do we only build ykllvm in the build target? I assume so.

That being the case `CARGO_BUILD_JOBS` seems best?

So if I run `cargo -j <blah>` cargo sets this var, and we pick it up. Correct?

Reply:
Side-note: `NUM_JOBS` is an awfully generic and clash-inviting variable name :) Not that we can do anything about it.

Reply:
Fixed in https://github.com/ykjit/yk/pull/888/commits/c8a1126aea25a64f03f7090b795671fecf8b7af3. Even better, cargo seems to *always* set this: if the user doesn't set `CARGO_BUILD_JOBS` then `NUM_JOBS` is set to the number of CPUs. So we don't even need the `num_cpus` crate anymore!

This will require some squashing.